frontend/public/locales/en/status.json

```diff
@@ -123,7 +123,7 @@
         "date-of-birth-is-past": "Date of birth must be in the past",
         "date-of-birth-is-past-valid": "Date of birth too far in the past",
         "date-of-birth-month-required": "Date of birth must include a month",
-        "date-of-birth-valid": "Day must be valid for the given month and year",
+        "date-of-birth-valid": "Selected date must be a valid calendar date.",
```

Comment:
This error message shouldn't differ from the rest of the application, unless a different error message was officially provided.